### ISSUE TITLE
heroku push doesn't recognise the --keys option

### ISSUE THREAD

**Issue by ideasasylum** [thekompanee/chamber#60]
## Current Behavior
The `heroku push` command doesn't recognise the keys option

### Steps to Reproduce

Using the instructions on the wiki page: https://github.com/thekompanee/chamber/wiki/CLI-chamber-heroku-push

`chamber heroku push --keys --app=myapp`

Results in the following error:

```
ERROR: "chamber heroku push" was called with arguments ["--keys"]
Usage: "chamber heroku push"
```

## Expected Behavior

It would push the private key to Heroku

## Environment

Gem version: 2.12.3

## Workaround

`heroku config:set --app="yourapp" CHAMBER_KEY="$(cat .chamber.pem)"`

[as noted elsewhere in the wiki](https://github.com/thekompanee/chamber/wiki/Loading-Keys-From-Environment-Variables#heroku)

**Comment by jfelchner** [comment]
@ideasasylum thanks for bringing this to my attention!  I'll get this knocked out next weekend!

**Comment by jfelchner** [comment]
@ideasasylum sorry for the wait.  Hopefully it was worth it. ;)

Merged in 4c7bb3f
